src/shell.js

```diff
@@ -279,9 +279,9 @@ if (ENVIRONMENT_IS_NODE) {
 
 } else
 #endif // ENVIRONMENT_MAY_BE_NODE
-#if ENVIRONMENT_MAY_BE_SHELL || ASSERTIONS
-if (ENVIRONMENT_IS_SHELL) {
 
+if (ENVIRONMENT_IS_SHELL) {
```

Comment:
If not `ENVIRONMENT_MAY_BE_SHELL` then we don't need this line, so I think code size was smaller before?